src/components/pages/Modal/NewTopicModalContainer.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import RenderNewTopicModal from './RenderNewTopicModal';
+
+const NewTopicModalContainer = () => {
+  return <>{<RenderNewTopicModal />}</>;
```

Comment:
Does RenderNewTopicModal need to be wrapped in a fragment as well as curly braces?